receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -92,7 +92,7 @@ metrics:
     enabled: true
     description: "Node CPU utilization"
     warnings:
-      if_enabled: "WARNING: This metric will be disabled in a future release. Use metric k8s.node.cpu.usage instead."
+      if_enabled: "This metric will be disabled in a future release. Use metric k8s.node.cpu.usage instead."
```

Comment:
Leaving the warning is ok. Do we have to remove it?

Reply:
The generated code adds a `[WARNING]` anyways in the log and hence we can remove this. Right now the log contains a `WARNING` twice :).

See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/internal/metadata/generated_metrics_test.go#L66